spool_3/kitten_spec.rb

```diff
@@ -18,31 +18,31 @@ class Kitten
 
   describe "feed" do
     it "returns true when fed ketchup on fish" do
-      expect(kitten.feed('ketchup on fish')).to be_true
+      expect(kitten.feed('ketchup on fish')).to be_truthy
```

Comment:
These specs all still want to `be_true` and `be_false` - I'm explicitly wanting the implementation of `feed` to return a boolean.